app/models/revision.rb

```diff
@@ -19,6 +19,11 @@ class Revision < ApplicationRecord
   after_save :inline_hashtags
   before_create :setup
 
+  scope :published, -> { where(status: 1) }
+  scope :weekly, -> { where("timestamp > ?", (Time.now - 7.days).to_i) }
```

Comment:
Oh, i'm sorry i should've noticed this before. This is OK and I can merge if you like, but in sense, "weekly" seems to mean "by week" rather than "the past week". Maybe the scope should be called `past_week`, OR, it should be `weekly` but then accept a parameter like starting date? I dunno, what do you think? It's a small semantic thing but if people begin using `.weekly` in their code they might think it means something a little different than what it is. Can scopes accept params? Or should we call it `past_week` `past_year` and such? 

Thanks Cess! This is really cool.

Reply:
Aha! I see bad naming I did there..I am not really sure about the scope accepting params and it seems complicated so I will just add  `past`. Thanks for that.